endstate_correction/simulation/base.py

```diff
@@ -125,13 +126,26 @@ def start(self):
                     _seed_velocities(_get_masses(self.simulation.system))
                 )
 
-        # define DCDReporter
+        # define HDF5Reporter
         self.simulation.reporters.append(
-            DCDReporter(
+            HDF5Reporter(
                 trajectory_file,
                 self.protocol.restart_interval,
             )
         )
         # perform sampling
         self.simulation.step(self.protocol.n_integration_steps)
         self.simulation.reporters.clear()
+
+    def get_trajectory(self) -> mdtraj_trajectory:
+        traj = mdtraj.load_hdf5(self._traj_file)
+        if self.env == "waterbox":
+            traj.image_molecules()
+        return traj
```

Comment:
can you comment on the motivation for `get_trajectory` and `get_xyz`?

Reply:
So the `get_xyz` method is used so that one could do
```
        sim = ec.get_simulation()
        traj = ec.get_xyz()
        fep_protocol = Protocol(
            method="NEQ",
            direction="bidirectional",
            sim=sim,
            trajectories=[traj, traj],
            nr_of_switches=5,
            neq_switching_length=10,
        )
```
The get_trajectory is just one layer up such that if the user want more, they could just get the MDTraj object.